components/cdc/tests/integrations/test_cdc.rs

```diff
@@ -968,3 +968,161 @@ fn test_cdc_resolve_ts_checking_concurrency_manager() {
     event_feed_wrap.replace(None);
     suite.stop();
 }
+
+#[test]
+fn test_cdc_1pc() {
+    let mut suite = TestSuite::new(1);
+
+    let req = suite.new_changedata_request(1);
+    let (mut req_tx, _, receive_event) = new_event_feed(suite.get_region_cdc_client(1));
+    block_on(req_tx.send((req, WriteFlags::default()))).unwrap();
+    let event = receive_event(false);
+    event.events.into_iter().for_each(|e| {
+        match e.event.unwrap() {
+            // Even if there is no write,
+            // it should always outputs an Initialized event.
+            Event_oneof_event::Entries(es) => {
+                assert!(es.entries.len() == 1, "{:?}", es);
+                let e = &es.entries[0];
+                assert_eq!(e.get_type(), EventLogType::Initialized, "{:?}", es);
+            }
+            other => panic!("unknown event {:?}", other),
+        }
+    });
+
+    // Sleep a while to make sure the stream is registered.
```

Comment:
why?

Reply:
I copy it from the old code. I remove it now.